### DISCUSSION TITLE
Welcome to meshinfo Discussions!

### DISCUSSION THREAD

**Discussion by kevinelliott** [MeshAddicts/meshinfo#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by quimnut** [comment]
I'm still chipping away at my mqtt protobufs thing.

Your SVM discord link has expired, was going to jump on and say g'day :)
